src/metabase/driver/sql_jdbc/sync/describe_database.clj

```diff
@@ -32,6 +32,23 @@
    (fn [^ResultSet rs]
      #(.getString rs "TABLE_SCHEM"))))
 
+(defn syncable-schemas-for-db
+  "Returns a collection of syncable schemas for a db if exists."
+  [database]
+  (let [driver                 (:engine database)
+        schema-filter-prop     (driver.u/find-schema-filters-prop driver)
+        schema-filter-patterns (when (and (some? schema-filter-prop) (some? database))
+                                 (driver.s/db-details->schema-filter-patterns (:name schema-filter-prop) database))]
```

Comment:
This looks like these lines duplicate the 1-arity version of `metabase.driver.sync/db-details->schema-filter-patterns`

Reply:
Actually I have already created a multimethod for this entire function. 
`driver/syncable-schemas`
see https://github.com/metabase/metabase/blob/f0b764e092b1212f186880dd5284842a5c8a1529/src/metabase/driver/sql_jdbc.clj#L188-L197

Reply:
oh nice!